zoomus/__init__.py

```diff
@@ -7,4 +7,4 @@
 
 
 __all__ = ["API_VERSION_1", "API_VERSION_2", "ZoomClient"]
-__version__ = "1.1.3"
+__version__ = "1.1.4"
```

Comment:
Can I ask you revert this change -- I update the versions when I make the official releases (i.e. this will be 1.1.6 when I get to it).

Reply:
fixed.